vimeo-networking/src/main/java/com/vimeo/networking/utils/VimeoNetworkUtil.java

```diff
@@ -71,7 +72,8 @@ public static Gson getGson() {
      */
     public static GsonBuilder getGsonBuilder() {
         // Example date: "2015-05-21T14:24:03+00:00"
-        return new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+        return new GsonBuilder().registerTypeAdapterFactory(new Stag.Factory())
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
```

Comment:
No, we shouldn't if we are annotating every model, but right now certain things aren't yet annotated and use reflection, namely, every single list (BaseResponseList) since we haven't solved for abstract types... yet. Leaving this in once those are working certainly does not hurt us.